src/microbe_stage/FloatingChunk.cs

```diff
@@ -106,6 +106,8 @@ public class FloatingChunk : RigidBody, ISpawned, ISaveLoadedTracked
     /// </summary>
     public string DamageType { get; set; } = "chunk";
 
+    public new string Name { get; set; } = string.Empty;
```

Comment:
Shouldn't be overriden, instead use a more explicit name like `ChunkName`.

Reply:
PerformanceMetrics is a Control